CMS-Enterprise/easi-app
# [NOREF] Add OHEI to component/office list

# NOREF

## Description

- Replaces `OBRHI` (Office of Burden Reduction and Health Informatics) with `OHEI` (Office of Healthcare Experience and Interoperability) in the list of CMS Offices and Divisions on the frontend
- Updates snapshot tests
- [x] `TODO` Adds a migration to the `./migrations/` folder that updates all instances of `OBRHI` to `OHEI`, and all instances of `"Office of Burden Reduction and Health Informatics"` with `"Office of Healthcare Experience and Interoperability"` where CMS Components are selected
> [!IMPORTANT]
> ⬆️ This `TODO` needs some research. Do we only store CMS Component acronyms on the backend? Where do we store acronyms? What all needs to be updated?



This should allow users to select this component, as well as automatically handle any logic that uses this array (like getAcronymForComponent)
https://github.com/CMS-Enterprise/easi-app/blob/ce7cc2c35576c661288685c519b49695c676055a/src/constants/enums/cmsDivisionsAndOffices.ts#L102-L105
https://github.com/CMS-Enterprise/easi-app/blob/ce7cc2c35576c661288685c519b49695c676055a/src/utils/systemIntake.ts#L27-L38

## How to test this change

- Create a new system intake
- Ensure that `OHEI` shows up as a selectable office when adding new contacts on the first page of the intake form

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
